package.json

```diff
@@ -178,6 +179,7 @@
     "eslint-plugin-jsx-a11y": "5.1.1",
     "eslint-plugin-react": "7.9.1",
     "eslint-scope": "3.7.1",
+    "fast-deep-equal": "^2.0.1",
```

Comment:
@plbabin Since it'll be used in `Table.Body`, we'll need to add this to `dependencies`, not `devDependencies` 😊